client/my-sites/customer-home/components/celebrate-launch-modal.jsx

```diff
@@ -25,21 +26,20 @@ function CelebrateLaunchModal( { setModalIsOpen, site, allDomains } ) {
 	const hasCustomDomain = customDomains.length > 0;
 	const hasEnTranslation = useHasEnTranslation();
 	const siteDomain = hasCustomDomain ? customDomains[ 0 ].domain : site.slug;
-	useEffect( () => {
-		// remove the celebrateLaunch URL param without reloading the page as soon as the modal loads
-		// make sure the modal is shown only once
-		window.history.replaceState(
-			null,
-			'',
-			omitUrlParams( window.location.href, 'celebrateLaunch' )
-		);
```

Comment:
This component shouldn't change the URL parameters. Let's leave it [to who called it](https://github.com/Automattic/wp-calypso/pull/104868/files#diff-ca010e8d1120fac2503dd02bfd901c378f6eb8c3ce74a9b5a26778f9a3dc8340R112-R119).